openstates/sc/people.py

```diff
@@ -88,11 +111,12 @@ def scrape(self, chamber, term):
                             '3rd V.C.': 'third vice-chair',
                             'Ex.Officio Member': 'ex-officio member',
                             'Chairman': 'chairman'}[role]
+
+                    person.extras['committee'] = committee
```

Comment:
Looks like `Person.add_membership` might be more appropriate here: https://github.com/opencivicdata/pupa/blob/master/pupa/scrape/popolo.py#L101